.eslintrc

```diff
@@ -38,13 +41,33 @@
     "jsx-quotes": ["error", "prefer-double"],
     "no-console": "warn",
     "react/forbid-prop-types": "off",
-    "react/destructuring-assignment": "off",
     "react/sort-prop-types": ["error"],
     "no-unused-vars": ["error", { "ignoreRestSiblings": true }],
     "padding-line-between-statements": ["error",
       { "blankLine": "always", "prev": ["const", "let", "var"], "next": "*"},
       { "blankLine": "any", "prev": ["const", "let", "var"], "next": ["const", "let", "var"]}
-    ]
+    ],
+
+    "no-else-return": ["error", { "allowElseIf": false }],
+    "no-nested-ternary": "error",
+    "prefer-arrow-callback": "error",
+    "arrow-parens": [2, "always"],
```

Comment:
looks like this is conflicting with sonar cloud rule - 

**Braces and parentheses should be used consistently with arrow functions**,

defined at  rule https://sonarcloud.io/organizations/folio-org/rules?open=javascript%3AS3524&rule_key=javascript%3AS3524

Reply:
Thanks for noticing, changed the rule

Reply:
Welcome! There are more than 250 rules defined in sonar cloud - https://sonarcloud.io/organizations/folio-org/rules?languages=js
Should we have an equivalent eslint config file on each our repositories?

Reply:
@Terala-Priyanka I think so, I'll create additional tickets for our other modules to use the same rules